test/unit/specs/components/search-grid-filter.spec.js

```diff
@@ -81,15 +46,10 @@ describe('SearchGridFilter', () => {
         },
       },
       stubs: { VIcon: true },
+      pinia,
```

Comment:
Why doesn't this use `createTestingPinia` like the other test? Are both ways accomplishing the same thing?

Reply:
Because `createTestingPinia` works only with Vue 3 :( I've replaced it in other tests, too.
The solution mentioned in the testing docs does not add a global plugin to Vue 2, so we get the "cannot find the Symbol pinia injection" error when we use it in tests.

Reply:
Ahh what a shame! Can you add a blurb in the testing docs about how to add pinia?